README.md

```diff
@@ -63,6 +63,16 @@ Here's an example setting an error and an output:
       end
     end
 
+If you'd like the `fatal_error` to raise a `StandardError` immediately instead of bubbling up to the `Lev::Routine#errors` object, you must configure it:
+
+```ruby
+Lev.configure do |config|
+  config.raise_fatal_errors = true
+end
+```
+
+So if you `fatal_error(name: :is_blank)` it will raise `StandardError: "name is blank"`, or `fatal_error(thing: :is_broken, and: :messed_up)` it will raise `StandardError: "thing is broken - and messed up"`
```

Comment:
Joe can you update the README to describe that `raise_fatal_errors` can be overridden in a particular routine?